### PR TITLE
feat: import leaks directly on the browser

### PR DESCRIPTION
**Pull request by freitzzz**
screen

### CODE REVIEW

File: third_party/import-web-api/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+const leakFormSchema = {
+    context: new String(),
+    leakFile: new String(),
+    shareDate: new Date(),
+    leakers: new String(),
+    platforms: new String(),
+};
+
+function triggerImportLeak(leakForm) {
+    const cmd = `./import --database-path="${leaksDbFilePath}" --leak-path="${leakForm.leakFile}" --context="${leakForm.context}" --platforms="${leakForm.platforms}" --share-date="${leakForm.shareDate}" --leakers="${leakForm.leakers}" --notify-url="https://subscribeService/notify"`;
```
Comment (rutesantos4) [inline third_party/import-web-api/index.js:26]: `--notify-url="https://subscribeService/notify"` should come from config

File: third_party/import-web/src/app.d.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// See https://kit.svelte.dev/docs/types#app
+// for information about these interfaces
+declare global {
+	namespace App {
+		// interface Error {}
+		// interface Locals {}
+		// interface PageData {}
+		// interface Platform {}
```
Comment (rutesantos4) [inline third_party/import-web/src/app.d.ts:8]: Remove if not needed

File: third_party/import-web-api/index.js (hunk 1 above)
Comment (freitzzz) [reply third_party/import-web-api/index.js:26]: done

File: third_party/import-web/src/app.d.ts (hunk 2 above)
Comment (freitzzz) [reply third_party/import-web/src/app.d.ts:8]: boilerplate code from svelte-kit, let's not remove it

**Review by rutesantos4 — APPROVED** [review]

**Pull request merged**